### PR TITLE
feat: cvar to clamp rainbow frequency

### PR DESCRIPTION
**Pull request by Rushaway**

### CODE REVIEW

File: addons/sourcemod/scripting/GlowColors.sp [hunk 1]
```diff
@@ -67,8 +68,8 @@ public void OnPluginStart()
 	HookEvent("player_team", Event_ApplyGlowcolor, EventHookMode_Post);
 
 	g_Cvar_MinBrightness = CreateConVar("sm_glowcolor_minbrightness", "100", "Lowest brightness value for glowcolor.", 0, true, 0.0, true, 255.0);
-	
 	g_Cvar_PluginTimer = CreateConVar("sm_glowcolors_timer", "5.0", "When the colors should spawning again (in seconds)");
+	g_Cvar_MinRainbowFrequency = CreateConVar("sm_glowcolors_minrainbowfrequency", "1.0", "Lowest frequency value for rainbow glowcolors before auto-clamp.", 0, true, 0.1, true, 60.0);
```
Comment (maxime1907) [inline addons/sourcemod/scripting/GlowColors.sp:72]: its not 100.0 for max frequency?

Comment (Rushaway) [reply addons/sourcemod/scripting/GlowColors.sp:72]: In fact their is no limit for the max frequency at this moment..

Comment (maxime1907) [reply addons/sourcemod/scripting/GlowColors.sp:72]: oh okay so no need to put an upper boundary for min rainbow effect then

Comment (Rushaway) [reply addons/sourcemod/scripting/GlowColors.sp]: Sure, done.

**Pull request merged**